source/dlangbot/bugzilla.d

```diff
@@ -79,3 +81,42 @@ Issue[] getDescriptions(R)(R issueRefs)
         .sort!((a, b) => a.id < b.id)
         .release;
 }
+
+shared string bugzillaLogin, bugzillaPassword;
+
+Json apiCall(string method, Json[string] params)
+{
+    import vibe.stream.operations : readAllUTF8;
+    import dlangbot.utils : request;
+
+    string[string] urlParams = [
+        "method" : method,
+    ];
+    if (params)
+        urlParams["params"] = params.Json.toString();
```

Comment:
This `toString()` is where the unstable sorting comes from.

Reply:
Also `urlEncode`, but that's easy to avoid.